### DISCUSSION TITLE
Please Help

### DISCUSSION THREAD

**Discussion by ArturoHDZG** [domcloud/domcloud#47] [Q&A] [answered]
I need help. I accidentally deleted a necessary file for running my project and there is no way to upload it again. Webmin is not working, I have tried to upload it, it keeps working on the file upload and then nothing, no error message or anything. I also haven’t been able to connect remotely from VSCode, I enter the password and the connection also fails. Two days ago everything was working fine. I like the service, but it’s impossible to work like this, it doesn’t let me do anything with my files (no edit, no upload) only delete.

“Edit: While testing, Webmin shows me two errors: when I try to create a new folder it throws the error: “No space left on device” and if I want to create a file it throws the error: permission denied: $4. What does all this mean?”

**Comment by willnode** [comment, marked as answer]
I noticed that our server storage get full quickly. I have added some few more free space to alleviate it.

Please see https://github.com/domcloud/domcloud/issues/48
